packages/inertia-vue/src/form.js

```diff
@@ -77,9 +77,6 @@ export default data => {
 
       return this
     },
-    hasErrors() {
-      return Object.keys(form.errors).length > 0
-    },
```

Comment:
Okay, so, I get that we're not using this, but wouldn't this be a super handy method for userland code? I hate having to do this:

```js
Object.keys(form.errors).length > 0
```

Much better to do:

```js
form.hasErrors()
```

Reply:
Yeah, I already re-added it after talking it over with @pxlrbt 😅 Just didn't push it up yet because lunch & the "remember" problem I'm working on ;)

Reply:
Haha, understood. 👍